pytorch_blade/ci_build/build_pytorch_blade.sh

```diff
@@ -41,29 +37,24 @@ function pip_install_deps() {
     python3 -m pip install -r ${requirements} -f https://download.pytorch.org/whl/torch_stable.html
 }
 
-function bazel_build() {
-    python3 ../scripts/python/common_setup.py
-    rm -rf build && python3 setup.py develop;
-}
-
-function cmake_build() {
-    rm -rf build && python3 setup.py develop --cmake;
-}
-
 function ci_build() {
+    echo "DO TORCH_BLADE CI_BUILD"
     pip_install_deps
 
     if [ "$TORCH_BLADE_USE_CMAKE_BUILD" = "ON"  ]; then
-      cmake_build
+      extra_args="--cmake"
```

Comment:
nit: How long are we going to maintain both build system?

Reply:
Until bazel build system is stable enough. I should not be too long.